Exercise-2/src/components/Cards.jsx

```diff
@@ -0,0 +1,65 @@
+
+import { useState } from 'react';
+import cardsData from '../data/cardsData.json'
+import Header from './Header';
+
+const shuffleArray= (array) =>{
+    let shuffledArray = [...array];
+    for(let i=shuffledArray.length-1; i>0; i--){
```

Comment:
how about this one `return array.sort(() => Math.random() - 0.5); for shuffling array?
Also lets move this in helpers.
Also lets plz consider the formating of code, to enforce you can use eslint rule.